pkg/bootkube/bootkube.go

```diff
@@ -94,6 +95,8 @@ func NewBootkube(config Config) (*bootkube, error) {
 }
 
 func (b *bootkube) Run() error {
+	fmt.Println("Running temporary bootstrap control plane...")
```

Comment:
Sounds good, I was trying to think of a way to make these sprinkled print functions clear that they are part of a specific output experience, but I didn't see an obvious way to group it all. 

A simple function wrapper to rename these sounds fine.